src/EffectTransform.js

```diff
@@ -123,15 +123,17 @@ class EffectTransform {
      * @returns {Uint8ClampedArray} dst filled with the transformed color
      */
     static transformColor (drawable, color4b, dst, effectMask) {
```

Comment:
Since we only call this from one place, what do you think about simplifying even further? Maybe something like this:
```suggestion
     static transformColor (drawable, inOutColor) {
```

Reply:
Should I remove the `effectMask` parameter too?